lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -498,6 +499,10 @@ void SPIRVRegularizeLLVMBase::adaptStructTypes(StructType *ST) {
            STElemTyName.startswith("__sycl_internal::")) &&
           STElemTyName.endswith("::half"))
         ElemTyStr = "half";
+      if ((STElemTyName.startswith("cl::sycl::") ||
```

Comment:
Can we use `isSYCLBfloat16Type()` here? And also use the appropriate function for `half` if it is possible?

Reply:
Changed both in [23872fa](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1427/commits/23872fa2b64a4f82fb04d0b665236653a3750f2d)